config/environment.js

```diff
@@ -46,7 +46,8 @@ module.exports = function(environment) {
                         width: 363,
                         height: 242
                     },
-                    permissionLanguage: 'no_trademark'
+                    permissionLanguage: 'no_trademark',
```

Comment:
I would rather fix this by building this list from the preprint_provider endpoint rather than having it stored in this js. @hmoco thought this should be doable without causing a lot of trouble. We should explore that option.